docs/source/make_a_release.rst

```diff
@@ -179,3 +179,44 @@ Step-by-step
       rm -rf dist/* && \
       rmvirtualenv pynwb-${release}-release && \
       rmvirtualenv pynwb-${release}-install-test
+
+-------------
+Conda Release
+-------------
+
+In order to release a new version on conda, follow the steps below:
+
+1. Clone feedstock
+
+   .. code::
+
+      $ git clone git@github.com:conda-forge/pynwb-feedstock.git
+
+
+2. Create a new branch
+
+   .. code::
+
+      $ cd pynwb-feedstok
+      $ git checkout -b new_version
```

Comment:
Please, use the convention where the variable is set an re-used:

```
release = X.Y.Z
```

Indicating that this is version used when releasing to PyPI